src/utils/InputHandler.js

```diff
@@ -0,0 +1,21 @@
+import InputView from '../views/InputView.js';
+import OutputView from '../views/OutputView.js';
+
+const InputHandler = {
+  async getValidInput(promptMessage, parser, validator, errorCategory) {
+    let isNotValid = true;
+    let parsedInput;
+
+    while (isNotValid) {
+      const input = await InputView.readLineAsync(promptMessage);
+      parsedInput = parser ? parser(input) : input;
+
+      const validationResults = validator(parsedInput);
+      OutputView.printValidationResults(validationResults, errorCategory);
```

Comment:
utils 함수인데, OutputView를 알고있네요! 아래처럼 util화 할 수 있겠어요.

```suggestion
      onValidate(validationResults, errorCategory);
```

아니면 이 함수도 InputView 안에 들어가도 될 것 같아요~!

Reply:
오 유틸함수인데 OutputView와 결합도가 있다는 측면은 생각하지 못했습니다...! InputHandler 자체를 InputView 메서드로 만들거나 outputview를 사용하는 부분도 유틸화하는 디테일은 생각하지 못했었는데 좋은 피드백 감사합니다!😊